test/modules/feedback/components/toggleFeedback/ToggleFeedbackPanel.test.js

```diff
@@ -100,6 +103,24 @@ describe('ToggleFeedbackPanel', () => {
 			expect(element.shadowRoot.querySelectorAll(MapFeedbackPanel.tag)).toHaveSize(0);
 			expect(element.shadowRoot.querySelectorAll(GeneralFeedbackPanel.tag)).toHaveSize(1);
 			expect(element.shadowRoot.querySelector(GeneralFeedbackPanel.tag).onSubmit).toEqual(element._onSubmit);
+			expect(store.getState().modal.currentStep).toBe(1);
+		});
+	});
+
+	describe('when modal state property "currentStep" changes', () => {
+		it('updates the UI', async () => {
+			// arrange
+			const element = await setup({ modal: { ...initialState, steps: 2, currentStep: 0 } });
+			// act
+			const generalButton = element.shadowRoot.querySelector('#feedbackGeneralButton');
+			generalButton.click();
+
```

Comment:
As part of the 'act'-preperations i like this kind of pre-assert. Just to be save for future changes and regression.
```suggestion
expect(element.shadowRoot.querySelectorAll(GeneralFeedbackPanel.tag)).toHaveSize(1);
```